src/PostgREST/App.hs

```diff
@@ -292,19 +292,21 @@ jsonMT = "application/json"
 csvMT :: BS.ByteString
 csvMT = "text/csv"
 
+allMT :: BS.ByteString
+allMT = "*/*"
+
 jsonH :: Header
 jsonH = (hContentType, jsonMT)
 
 contentTypeForAccept :: Maybe BS.ByteString -> Maybe BS.ByteString
 contentTypeForAccept accept
-  | isNothing accept || hasJson = Just jsonMT
-  | hasCsv = Just csvMT
+  | isNothing accept || has allMT || has jsonMT = Just jsonMT
+  | has csvMT = Just csvMT
```

Comment:
How would this behave if both `text/csv` and `*/*` were specified? (My hope would be that csv would prevail because it is more specific.)